.vale/fixtures/OpenShiftAsciiDoc/MergeConflictMarkers/.vale.ini

```diff
@@ -0,0 +1,5 @@
+; Vale configuration file to test the `MergeConflictMarkers` rule
+StylesPath = ../../../styles
+MinAlertLevel = warning
```

Comment:
Error or warning? These markers can cause builds to fail if they come between Asciidoc syntax.

Reply:
If the rule is cast iron, I'd put it as an error. We deffo don't want to see these in the docs. Only errors will flag in the OCP vale-bot.

However, again, you would want to really check you don't introduce a load of false positives. You could actually run the rule against the whole repo to see what it produced. I'm on my holidays right now, back Tuesday! Off to Spain :-)

Reply:
AH, gosh, sorry for ruining the holiday vibes! No rush with this. Enoy the sun! Thanks for your help.